advanced_circuits_algorithms/QAA/QAA_tutorial.ipynb

```diff
@@ -158,13 +158,13 @@
     "$$-\\mathcal{R}_{B} = 2 |G\\rangle |1\\rangle \\langle 1|\\langle G| - \\mathbb{1},$$ \n",
     "\n",
     "which is a reflection about $|G\\rangle |1\\rangle$, because $-\\mathcal{R}_{B} |G\\rangle |1\\rangle = +1|G\\rangle |1\\rangle$ and $-\\mathcal{R}_{B} |B\\rangle |0\\rangle = -1|B\\rangle |0\\rangle$. \n",
-    "This transformation can be simply achieved by applying $X_{n+1}Z_{n+1}X_{n+1}$ to the (last) ancilla qubit. \n",
+    "This transformation can be achieved by applying $X_{n+1}Z_{n+1}X_{n+1}$ to the (last) ancilla qubit. \n",
     "This way, we obtain a minus sign whenever the ancilla is in the $|0\\rangle$ state. \n",
     "\n",
-    "__Implementation of $-\\mathcal{R}_{0}$__: We need to implement the transformation $|0, \\dots, 0\\rangle \\rightarrow -|0, \\dots, 0\\rangle$, while leaving all other states untouched. \n",
+    "__Implementation of $-\\mathcal{R}_{0}$__: We must implement the transformation $|0, \\dots, 0\\rangle \\rightarrow -|0, \\dots, 0\\rangle$, while leaving all other states untouched. \n",
     "To this end, we can flip all the qubits (using single-qubit $X$ gates), flipping the sign of $|11...1\\rangle$, and flipping the qubits back. \n",
     "Thus, the last operation left to explicitly define is flipping the sign of $|11...1\\rangle$. \n",
-    "This can be done with ancilla qubits.\n",
+    "This operation can be done with ancilla qubits.\n",
```

Comment:
`Thus, the last operation left to explicitly define is flipping the sign of $|11...1\\rangle$, which can be done with ancilla qubits`?